src/app/shared/services/mailing.js

```diff
@@ -0,0 +1,32 @@
+import * as emailjs from 'emailjs-com';
+
+export default class Mailing {
```

Comment:
Set a more descriptive name like 'MailingService'

Reply:
The other services got same naming, they are inside services folder!